### DISCUSSION TITLE
Rut Palmero @Getronics (Spain)

### DISCUSSION THREAD

**Discussion by rutpal** [Green-Software-Foundation/discussions#50] [Introductions]
My name is Rut, I'm a Project Manager and I'm part of the ESG Local Team @Getronics, where I'm in charge of engaging ourc olleagues on the definition of technological solutions so that our clients achieve their sustainability objectives, and to innovate in the way of doing so with a minimum environmental impact. I'm interested in getting in touch in people that is working in the fiels of IT sustainability, Green Software, etc, specially in Spain.
You can contact me on [LinkedIn
](https://www.linkedin.com/in/rutpal/?locale=en_US)

**Comment by NAMRATA-WOKE** [comment]
Hi @rutpal, glad you've connected with the community on GSF Discussions. Make sure you check out the [Welcome Discussion](https://github.com/orgs/Green-Software-Foundation/discussions/2) that has more activity to connect with like-minded practitioners. Did you know we have a [Meetup group in Madrid](https://www.meetup.com/gsf-madrid/)? It's worth joining that community, and if you're interested in organizing an event, you can reach out to @adamj89 who leads our community practice!
